community/events/12th-annual-users-meeting-2017.html

```diff
@@ -68,19 +188,200 @@ <h5 class="event-time">09:30 - 10:45</h5>
                 <h3 class="event-title">Workshops - Round 1</h3>
                 <ul class="event-workshop">
                     <li class="event-workshop-title"><span class="event-workshop-room">Room 1</span> - <a href="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/Using-OMERO/Using-OMERO-Workshop-Outline.pdf">Using OMERO</a></li>
-                    <li>Informal demonstrations of OMERO.clients, especially OMERO.web</li>
-                    <li>Informal demonstrations of web apps: figure, iviewer, webtagging etc.</li>
-                    <li>Informal demonstrations of other tools developed by the OME consortium e.g. mtools, ImageJ</li>
-                    <li>Discussions as requested on integrating your own tools: Java, Python etc.</li>
-                    <li>Discussions as requested on using/customizing training materials</li>
+                    <li>
+                        Informal demonstrations of OMERO.clients, especially
+                        OMERO.web
+                    </li>
+                    <li>
+                        Informal demonstrations of web apps: figure, iviewer,
+                        webtagging etc.
+                    </li>
+                    <li>
+                        Informal demonstrations of other tools developed by
+                        the OME consortium e.g. mtools, ImageJ
+                    </li>
+                    <li>
+                        Discussions as requested on integrating your own
+                        tools: Java, Python etc.
+                    </li>
+                    <li>
+                        Discussions as requested on using/customizing training
+                        materials
+                    </li>
                 </ul>
                 <ul class="event-workshop">
                     <li class="event-workshop-title"><span class="event-workshop-room">Room 2</span> - Working with OME Files and Bio-Formats (developer-aimed session)</li>
-                    <li>Overview of OME Model and Formats, Bio-Formats and OME Files</li>
-                    <li><a href="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/Introduction-to-Bio-Formats/#/">Introduction to working with Bio-Formats</a></li>
-                    <li><a href="http://downloads.openmicroscopy.org/presentations/2017/Users-Meeting/Workshops/OME-Files-C++/cpp-dundee-2017.pdf">Introduction to working with OME Files</a></li>
+                    <li>
+                        Overview of OME Model and Formats, Bio-Formats and OME 
```

Comment:
Huh - it looks fine in Safari but I just tried Firefox instead and it does indeed look different, although it's the items underneath for this session and OME.smuggler further down the page which don't match all the rest - this line has the same indent as the list for Room 1 etc.

I'll have a play and fix that later